google-cloud-spanner/src/test/java/com/google/cloud/spanner/GceTestEnvConfig.java

```diff
@@ -58,8 +58,11 @@ public class GceTestEnvConfig implements TestEnvConfig {
   private final SpannerOptions options;
 
   public GceTestEnvConfig() {
-    String projectId = System.getProperty(GCE_PROJECT_ID, "");
-    String serverUrl = System.getProperty(GCE_SERVER_URL, "");
+    // TODO: Revert back any changes to this file before merging.
+    // String projectId = System.getProperty(GCE_PROJECT_ID, "");
+    String projectId = "span-cloud-testing";
+    // String serverUrl = System.getProperty(GCE_SERVER_URL, "");
```

Comment:
It seems we forgot to clean this file?

Reply:
I'll clean it before merging. Have added a TODO here.